src/checkbox/index.js

```diff
@@ -27,11 +28,11 @@ class Checkbox extends PureComponent {
   }
 
   render() {
-    const { checked, indeterminate = false, onChange, disabled, readOnly, ...rest } = this.props
+    const { checked, indeterminate = false, onChange, disabled, readOnly, radio, ...rest } = this.props
 
     const isDisabled = disabled || readOnly
 
-    return <input type="checkbox" ref={refUpdater({ checked, indeterminate })} onChange={this.handleChange} disabled={isDisabled} {...rest} />
+    return <input type={radio ? 'radio' : 'checkbox'} ref={refUpdater({ checked, indeterminate })} onChange={this.handleChange} disabled={isDisabled} {...rest} />
```

Comment:
Added a radio now. Thought about using const Tag = radioSelect ? RadioButton : CheckBox but left that idea. Also noticed some bug for firefox which don't seem to be an issue any more. The outside click is since then rewritten so I guess it's sorted out since then?